docs/tutorial/overview.md

```diff
@@ -10,16 +10,15 @@ prodvec := map 16 (a <- avec, b <- bvec) { a * b }
 dot := reduce 4 (a, b <- prodvec) 0 { a + b }
 ```
```

Comment:
I would like to see more explanation about the language itself here. What is the semantics of `map` and `reduce`?